lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
Yes, nth occurred to me since that is what I am used to using in Erlang (though not Elixir). However, again, it seemed a bit 'technical'. My issue with `pick`, along with all my other sugegstions, is that it does not nicely capture the fact that we are selecting only 1 item from the list. (If I was to present someone with a row of chocolates (beers, whatever) and said `pick 3` I doubt I would get the result we are talking about here!)

If you are OK with `item` then I am convinced that is the best.